template.yaml

```diff
@@ -250,13 +250,14 @@ objects:
             * Adding additional operators to the cluster will increase these requirements depending on the operators chosen.
 
       4.  **Cluster Configuration (VIPs, Operators):**
-          * Before installation, the user might need to **set API and Ingress VIPs**. Proactively ask if they want to configure these.
+          * Before installation, the user might need to **set API and Ingress VIPs**. Only offer this for multi-node clusters with installer-managed networking, and only after hosts have been discovered (post-ISO boot) so that subnets are known.
```

Comment:
```suggestion
          * Before installation, the user might need to **set API and Ingress VIPs**. Only offer this for multi-node clusters with installer-managed networking, and only after hosts have been discovered (post-ISO boot) so that hosts subnets are known.
```